### ISSUE TITLE
Investigate options for enabling mentions in the visual tab in forums

### ISSUE THREAD

**Issue by epoirier** [myfossil/myfossil-theme#242]

**Comment by shanebp** [comment]
This does not seem possible.

**Comment by epoirier** [comment]
Here's a thread where people discuss how they made it work.  Looks like it's doable, but outside of the scope of this iteration of work.  https://buddypress.org/support/topic/mention-autosuggest-in-visual-editor/

**Comment by shanebp** [comment]
Yes, I saw that thread - it involves hacking ~3 core files in a standard install.  Not sure what would be necessary in the MF 'flavor'.
